### PR TITLE
Sync Internal changes

### PR DESCRIPTION
**Pull request by mingxingtan**
PiperOrigin-RevId: 344144225

### CODE REVIEW

File: efficientdet/keras/train_lib.py [hunk 1]
```diff
@@ -733,11 +739,21 @@ def train_step(self, data):
         tf.summary.image('input_image', images)
     with tf.GradientTape() as tape:
       if len(self.config.heads) == 2:
-        cls_outputs, box_outputs, seg_outputs = self(images, training=True)
+        cls_outputs, box_outputs, seg_outputs = util_keras.fp16_to_fp32_nested(
```
Comment (mingxingtan) [inline efficientdet/keras/train_lib.py:742]: It is better to use float32 for loss computing.

**Review by mingxingtan — COMMENTED** [review]
This PR is mostly about simple format changes.

**Review by fsx950223 — APPROVED** [review]

File: efficientdet/dataset/inspect_tfrecords.py [hunk 2]
```diff
@@ -81,7 +77,7 @@ def visualize(self):
     scale_image = tf.constant([0.229, 0.224, 0.225])
     scale_image = tf.reshape(scale_image, (1, 1, -1))
 
-    logging.info(f"Visualizing TfRecords {FLAGS.file_pattern} ....")
+    logging.info('Visualizing TfRecords %s', FLAGS.file_pattern)
```
Comment (kartik4949) [inline efficientdet/dataset/inspect_tfrecords.py:80]: @mingxingtan i thought using f strings is better way as of python >=3.6

Comment (mingxingtan) [reply efficientdet/dataset/inspect_tfrecords.py:80]: Hi @kartik4949, yes, f-strings are generally good, but logging prefers %s for the benefit of lazy evaluation.

f-string would cause pylint warning logging-format-interpolation, see: https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation?rq=1

Comment (kartik4949) [reply efficientdet/dataset/inspect_tfrecords.py:80]: Informative :) thanks
